pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -391,6 +394,14 @@ func (v *VolumeOptions) RunVolume(args []string) error {
 		return err
 	}
 
+	if v.List {
+		listingErrors := v.printVolumes(infos)
+		if len(listingErrors) > 0 {
+			return cmdutil.ErrExit
```

Comment:
I could move error printing to here but since `v.printVolumes` already prints the errors, I did not bother.